iOS/MMIWG2S/ViewControllers/ViewController2.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ViewController2.swift
+//  MMIWG2S
+//
+//  Created by Marvin Amaro on 8/23/20.
+//        Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import UIKit
+
+class ViewController2:UIViewController {
```

Comment:
Re comment from beginning: Let's try to just use one more abstract class that could be implemented 4 times with different parameters.